pom.xml

```diff
@@ -58,6 +58,20 @@
 			<version>${matsim.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.matsim.contrib</groupId>
+			<artifactId>drt</artifactId>
+			<version>${matsim.version}</version>
+			<scope>compile</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.matsim.contrib</groupId>
+			<artifactId>application</artifactId>
+			<version>14.0-SNAPSHOT</version>
```

Comment:
The application dependency is now duplicated. This one can be removed. Also, it uses a hard-coded version number instead of the property.